Week-7/Assignment-6/index.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * @function Queue a function to generate a Queue using two stacks
+ */
+function Queue() {
+    let s1 = [];
+    let s2 = [];
+
+    // Time Complexity - O(n), Ω(1), Space Complexity - O(1)
+    this.enqueue = function (value) {
+        if (s1.length > 100) throw new Error("Stack overflow!");
+        if (value < 1 || value > 100) throw new Error("Value out of bounds!");
+        
+        if (s2.length > 0) {
```

Comment:
Why do we need to do this ?
Please clarify.

Reply:
It was a constraint mentioned as part of requirement/ problem.